.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy
+on:
+  push:
+  pull_request:
+jobs:
+  build:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download source
+        uses: actions/checkout@v2
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/bundle
+            tmp/.htmlproofer
+          key: gem-${{ hashFiles('Gemfile.lock') }}
+          restore-keys: gem-
+      - name: Build website
+        run: |
+          mv docker-compose.ci.yml docker-compose.override.yml
+          docker-compose run --rm web /bin/bash -c "gem install html-proofer && (bundle check || bundle install --jobs=3)"
```

Comment:
Maybe this should be added to docker-compose.yml to be useful locally?

Reply:
Rather than that I would add it to the Gemfile.

I don't know why html-proofer is not part of the Gemfile. I mostly translated the .travis.yml

Reply:
I probably didn't add it because I have it globally installed locally anyways... ;)